src/Configuration.cpp

```diff
@@ -64,9 +64,25 @@ void Configuration::begin()
 void Configuration::setModuleMode(ModuleMode f)
 {
   currentMode = f;
+  byte oldMode = storage->readEEPROM(0);
   storage->writeEEPROM(0, f);
```

Comment:
This overwrites any flags like heartbeat.

Reply:
Thank you! I missed this.  I had it in as a stepping stone as I worked through the changes.  It then became redundant and whilst I took out all of its use, I forgot to take it out!

Reply:
Not a problem if you leave eeprom[0] as currentMode and store the flags in eeprom[4].

Reply:
I think that we could have a discussion about how we organise the lower eeprom addresses. We do not have to be constrained by the way in which Duncan did it for CBUS.